### DISCUSSION TITLE
Drawer

### DISCUSSION THREAD

**Discussion by Febakke** [digdir/designsystemet#1819] [Components]
### Related issues or relevant information?

- https://design.entur.no/komponenter/layout-og-flater/drawer

Issue: https://github.com/digdir/designsystemet/issues/221

**Comment by Barsnes** [comment]
Ein `Drawer` oppfører seg ofte som ein `Modal`. Den vil fange fokus, og du kan ikkje trykke på anna innhold på sida.
Shadcn har både `Drawer` og `Sheet`, der drawer kjem opp frå bunnen, og sheet fra sida. Eg føler ein drawer kan være begge deler
https://ui.shadcn.com/docs/components/sheet
https://ui.shadcn.com/docs/components/drawer

Eg lurer på om det hadde vært mulig å bruke native `dialog`-elementet til dette, som me gjer for `Modal`. 🤔
Fra MDN, så høyrest det ut som noko som kan funke:
> The HTML <dialog> element is used to create both modal and non-modal dialog boxes. Modal dialog boxes interrupt interaction with the rest of the page being inert

Den kjem med innebygd tilgjengeligheit både for fokus og tastatur.
[Dialog på MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog)

**Comment by mimarz** [comment]
Suggested implementation and features from community: https://github.com/digdir/designsystemet/pull/2203

**Comment by ArildR82** [comment]
Hei, jeg hadde behov for ett drawer komponent og laget ett som baserer seg på tabs komponentet i designsystemet. Deler komponentet her, så får vi se om det kommer til noen nytte.  

https://www.figma.com/design/jlUnAwGkGH6U6m42spD1tm/Designsystemet---Core-UI-Kit-(Community)?node-id=12584-15585&m=dev

**Comment by mrosvik** [comment]
Lukker denne siden den nå er laget i Designsystemet.
